### DISCUSSION TITLE
Using AdobeRGB

### DISCUSSION THREAD

**Discussion by omaximenco** [mono/SkiaSharp#1663] [Q&A]
**Description**

There is no way to use the AdobeRGB color space defined in Skia 

**Code**

```csharp
var surface = SKSurface.Create(new SKImageInfo(width, height, 
    original.ColorType, original.AlphaType, ??? AdobeRGB color space ???));
```

**Expected Behavior**

AdobeRGB color space is defined in Skia, so I guess it should be defined in SkiaSharp?
SKColorSpace.CreateSrgb() is available - why not to make SKColorSpace.CreateAdobeRgb()?

**Basic Information**

- Version with issue:  2.80.2

**Comment by mattleibow** [comment]
I see there is no nice helper, but there is a way:

```csharp
SKColorSpace.CreateRgb(SKColorSpaceTransferFn.Srgb, SKColorSpaceXyz.AdobeRgb);
```

Both of the types used in there have a few other variants.
